introduction.md

```diff
@@ -1,2 +1,5 @@
 * This is an intro.
 * My favorite Code School path is Electives.
+* On my way to becoming a GitHub Master. I love Ruby and Javascript.
+* Ruby on Rails
+* Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?